frontend/src/components/dashboard/geo_location/MapButtons.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <VItemGroup mandatory>
+        <VItem
+            v-for="button in buttons"
+            :key="button.label"
+        >
+            <ButtonComponent
+                :type="button.type"
+                :label="button.label"
+                @change="changeParameter"
+            />
+        </VItem>
+    </VItemGroup>
+</template>
+
+<script>
+    import ButtonComponent from "./ButtonComponent";
+    import {mapActions} from 'vuex';
+    import {CHANGE_SELECTED_PARAMETER} from "@/store/modules/geo_location/types/actions";
+
+    export default {
+        name: "MapButtons",
+        components: {
+            ButtonComponent
+        },
+        data() {
+            return {
+                buttons: [
+                    {
+                        label: 'Visitors',
+                        type: 'visitors',
+                    },
+                    {
+                        label: 'Sessions',
+                        type: 'sessions'
+                    },
+                    {
+                        label: 'Bounce Rate',
+                        type: 'bounce_rate'
+                    },
+                ]
+            };
+        },
+        methods: {
+            ...mapActions('geo_location', {
+                changeSelectedParameter: CHANGE_SELECTED_PARAMETER
```

Comment:
See comment in button component, or get active value and dispatch action here or do this in button component, but don't split vuex usage among both